framework/src/org/checkerframework/framework/type/DefaultTypeHierarchy.java

```diff
@@ -1031,6 +1031,10 @@ protected boolean visitWildcardSubtype(
                 // TODO: wildcard capture conversion
                 // Two unannotated uses of reference-equal wildcard types are the same type
                 return true;
+            } else {
+                AnnotatedWildcardType superWildcard = (AnnotatedWildcardType) supertype;
```

Comment:
Some comment about the logic for this branch would be great. Once the TODO on line 1031 about wildcard captures is addressed, is this case still necessary?
Why is the second condition the way it is and not `isSubtype(supertype.getSuperBound(), subWildcard.getSuperBound())`, which would ensure that the subtype range is smaller than the supertype range?